app/views/application/index.html.erb

```diff
@@ -1,65 +1,46 @@
-<%= javascript_include_tag 'index' %>
-<%= render "header" %>
-
-  <div class="row">
-    <div class="col-md-2">
-      <div class="row">
-        <h5>Nearby Locations</h5>
-      </div>
-      <div class="row">
-        <%= link_to 'Show All Locations', locations_path%>
-      </div>
-      <div class="row">
-        <div id="nearby-locations-sidenav" class="sidenav">
-          <% @locations_full.each do |location| %>
-            <div id="location-card" class="card">
-              <div class="card-body">
-                <h5><%=link_to location.name, location.websiteURL, target: "_blank"%></h5>
-                <p class="card-text" style="font-size: 12px;"><h5><%= number_with_precision(hav_distance([location.latitude.to_f, location.longitude.to_f], @current_location), precision: 2) %> mi</h5></p>
-              </div>
-            </div>
-            <br>
-            <%= content_tag :div, class: "j-location_info", data: {loc: location} do %><% end %>
-          <% end %>
-        </div>
-      </div>
+<div class="header">
+  <div class="title">
+    <%= image_tag("sfbbo-logo.png", class: "logo") %>
+    <div class="name">BAY TRAIL BIRD MAPPING</div>
```

Comment:
I think it's generally considered better/more accessible to avoid all caps and use CSS instead (text-transform: uppercase), since the capitalization is stylistic here.

Reply:
I don't think that we should add a CSS property since that increases file size and forces additional styling to load on the element.